python/ray/data/_internal/plan.py

```diff
@@ -335,9 +335,16 @@ def execute(
         Returns:
             The blocks of the output dataset.
         """
+        context = DatasetContext.get_current()
+        if not ray.available_resources().get("CPU"):
+            logger.get_logger(log_to_stdout=context.enable_auto_log_stats).warning(
```

Comment:
This message is not stats, it seems `enable_auto_log_stats` is not an applicable flag. 
Also, should we by default not log to stdout?

Reply:
Thanks for the catch, removed here. Yes, by default, the logger will log to stdout. There are some cases within the `DatasetStats` where we wanted to avoid cluttering stdout with long outputs, since they are always written to the dataset log file. (This is not one of those cases, as we definitely want the warning in stdout)